frontend/package.json

```diff
@@ -41,6 +40,7 @@
     "postcss": "^8.4.33",
     "prettier": "3.2.4",
     "pretty-quick": "^4.0.0",
+    "sass": "^1.70.0",
```

Comment:
Are there any reason to use `scss` sass nowadays really?

Reply:
Nested CSS mainly.

Reply:
Nested CSS is supported natively in CSS https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting

Reply:
Not great browser support yet: https://caniuse.com/css-nesting

Reply:
So all the browsers our react code will run on?